src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -112,7 +103,7 @@ While it is possible to install those charts separately, we strongly recommend u
       enabled: true
   ```
 
-4. Make sure everything is configured properly in the chart by running the following command. Notice that we are specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
+4. Make sure everything is configured properly in the chart by running the following command. Notice that we're specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
 
   ```shell
     helm upgrade --install newrelic newrelic/nri-bundle \
```

Comment:
We change this so that it's consistent with the guided install helm release name of `newrelic-bundle`.

```
helm upgrade --install newrelic-bundle newrelic/nri-bundle \
```